chapters/BP_044_en.md

```diff
@@ -0,0 +1,67 @@
+## Use event delegations
+
+### Identifiers
+
+| GreenIT | V2  | V3  |  V4  |
+|:-------:|:---:|:---:|:----:|
+|   41    | 46  | 44  |      |
+
+### Categories
+
+|    Life cycle     |    Tiers    |         Responsible          |
+|:-----------------:|:-----------:|:----------------------------:|
+| 3. Implementation | User/Device | Software Architect/Developer |
+
+### Indications
+
+|      Priority      | Implementation difficulty  | Ecological impact |
+|:------------------:|:--------------------------:|:-----------------:|
+|         3          |             3              |        4          |
+
+|                      Saved resources                      |
+|:---------------------------------------------------------:|
+|                      Processor / RAM                      |
+
+### Description
+
+Events delegation avoids overloading browser memory by instantiating a single listener for several DOM (Document Object Model) elements.
+
+### Examples
+
+The DOM element whose ID is "t" is declared as the delegate.
+
+It intercepts the events of all of its children.
+
+```html
+<head>
```

Comment:
This is code is not accessible. We must add focusable element inside of the `table`